### ISSUE TITLE
Unable to open admin tabs in 1.0.113

### ISSUE THREAD

**Issue by alex-hempel** [Eugeny/terminus#2800]
When trying to open any admin tab in 1.0.113 via right-click and admin tab, a tab briefly opens and immediately closes again.

Windows 10 Version 2004

![2020-07-20_14-40-29](https://user-images.githubusercontent.com/6581307/87900270-3ec21280-ca97-11ea-8121-2709b9d4a3d7.gif)

**Comment by Eugeny** [comment]
Please capture DevTools console logs at the time of the issue: https://github.com/Eugeny/terminus/wiki/Debug-logs

**Comment by alex-hempel** [comment]
I got these error messages when trying to open a PowerShell admin tab:

```log.service.ts:56 [terminal] Using session options: {command: "C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\extras\UAC.exe", args: Array(2), env: {…}, runAsAdministrator: true}command: "C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\extras\UAC.exe"args: Array(2)0: "powershell.exe"1: "-nologo"length: 2__proto__: Array(0)env: TERM: "cygwin"__proto__: constructor: ƒ Object()__defineGetter__: ƒ __defineGetter__()__defineSetter__: ƒ __defineSetter__()hasOwnProperty: ƒ hasOwnProperty()__lookupGetter__: ƒ __lookupGetter__()__lookupSetter__: ƒ __lookupSetter__()isPrototypeOf: ƒ isPrototypeOf()propertyIsEnumerable: ƒ propertyIsEnumerable()toString: ƒ ()valueOf: ƒ valueOf()toLocaleString: ƒ toLocaleString()get __proto__: ƒ __proto__()set __proto__: ƒ __proto__()runAsAdministrator: true__proto__: constructor: ƒ Object()__defineGetter__: ƒ __defineGetter__()__defineSetter__: ƒ __defineSetter__()hasOwnProperty: ƒ hasOwnProperty()__lookupGetter__: ƒ __lookupGetter__()__lookupSetter__: ƒ __lookupSetter__()isPrototypeOf: ƒ isPrototypeOf()propertyIsEnumerable: ƒ propertyIsEnumerable()toString: ƒ ()valueOf: ƒ valueOf()toLocaleString: ƒ toLocaleString()get __proto__: ƒ __proto__()set __proto__: ƒ __proto__()
/C:/Users/alex.hempel/AppData/Local/Programs/Terminus/resources/app.asar/dist/[object%20Module]:1 GET file:///C:/Users/alex.hempel/AppData/Local/Programs/Terminus/resources/app.asar/dist/[object%20Module] net::ERR_FILE_NOT_FOUND
xtermFrontend.ts:117 Could not resize xterm Error: This API only accepts integers
    at eval (xterm.js:1)
    at Array.forEach (<anonymous>)
    at e._verifyIntegers (xterm.js:1)
    at e.resize (xterm.js:1)
    at XTermFrontend.resizeHandler (xtermFrontend.ts:113)
    at eval (xtermFrontend.ts:158)
    at ZoneDelegate.invokeTask (C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:421)
    at Zone.runTask (C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:188)
    at ZoneTask.invokeTask (C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:503)
    at ZoneTask.invoke (C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:492)
    at timer (C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:3034)
    at sentryWrapped (C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\dist\sentry.js:1882)
XTermFrontend.resizeHandler @ xtermFrontend.ts:117
eval @ xtermFrontend.ts:158
ZoneDelegate.invokeTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:421
Zone.runTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:188
ZoneTask.invokeTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:503
ZoneTask.invoke @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:492
timer @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:3034
sentryWrapped @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\dist\sentry.js:1882
setTimeout (async)
(anonymous) @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\dist\sentry.js:1975
scheduleTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:3055
ZoneDelegate.scheduleTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:407
Zone.scheduleTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:231
Zone.scheduleMacroTask @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:254
scheduleMacroTaskWithCurrentZone @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:1183
(anonymous) @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:3068
proto.<computed> @ C:\Users\alex.hempel\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:1501
eval @ xtermFrontend.ts:158
```

**Comment by mesa123123** [comment]
I'm going to +1 This is what I'm getting on the log files

```
[terminal] Using session options: {command: "C:\Users\bowma\AppData\Local\Programs\Terminus\resources\extras\UAC.exe", args: Array(2), env: {…}, runAsAdministrator: true}command: "C:\Users\bowma\AppData\Local\Programs\Terminus\resources\extras\UAC.exe"args: (2) ["powershell.exe", "-nologo"]env: {TERM: "cygwin"}runAsAdministrator: true__proto__: Object
/C:/Users/bowma/AppData/Local/Programs/Terminus/resources/app.asar/dist/[object%20Module]:1 
---------------------------------------------
GET file:///C:/Users/bowma/AppData/Local/Programs/Terminus/resources/app.asar/dist/[object%20Module] net::ERR_FILE_NOT_FOUND
xtermFrontend.ts:117 
-----------------------------------------------
Could not resize xterm Error: This API only accepts integers
    at eval (xterm.js:1)
    at Array.forEach (<anonymous>)
    at e._verifyIntegers (xterm.js:1)
    at e.resize (xterm.js:1)
    at XTermFrontend.resizeHandler (xtermFrontend.ts:113)
    at eval (xtermFrontend.ts:158)
    at ZoneDelegate.invokeTask (C:\Users\bowma\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:421)
    at Zone.runTask (C:\Users\bowma\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:188)
    at ZoneTask.invokeTask (C:\Users\bowma\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:503)
    at ZoneTask.invoke (C:\Users\bowma\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:492)
    at timer (C:\Users\bowma\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\zone.js\dist\zone.js:3034)
    at sentryWrapped (C:\Users\bowma\AppData\Local\Programs\Terminus\resources\app.asar\dist\sentry.js:1882)
```

System info:
Win10 - Version 1909 (OS Build 18363.959)

**Comment by alex-hempel** [comment]
Still happening for me in 1.0.114.

**Comment by Eugeny** [comment]
See if `UAC.exe` crashes if you double-click it in `%AppData%\Local\Programs\Terminus\resources\extras\`

**Comment by alex-hempel** [comment]
It does: 

`---------------------------
UAC.exe - System Error
---------------------------
The code execution cannot proceed because VCRUNTIME140_1D.dll was not found. Reinstalling the program may fix this problem. `

**Comment by Eugeny** [comment]
Please try this nightly: https://ci.appveyor.com/project/Eugeny/terminus/build/artifacts

**Comment by alex-hempel** [comment]
Looks like that build failed.

**Comment by Eugeny** [comment]
Direct link to the download: https://ci.appveyor.com/api/buildjobs/ywk4vajlgghr9dqv/artifacts/dist%2Fterminus-1.0.116-nightly.0-setup.exe

**Comment by fluxtendu** [comment]
1.0.116 nightly partially corrected the bug on my side, tabs stay open and no issues with Cmder, CMD (stock) and PowerShell
but i get new issues with others:

**CMD (clink)** 
```
*** ERROR: Unknown verb -- 'inject '
Clink v0.4.9 [git:2fd2c2] Copyright (c) 2012-2016 Martin Ridgers
http://mridgers.github.io/clink

Usage: <verb> <verb_options>

  Verbs:
  inject   Injects Clink into a process.
  autorun  Manage Clink's entry in cmd.exe's autorun.
  set      Adjust Clink's settings.
           ('<verb> --help' for more details)
```

**PowerShell Core**

No error but uppercase add a "à" before character. For example if I write "SaMplE", I get "àSaàMplàE"

**WSL / Ubuntu**

`/bin/bash: -d: command not found`
and freeze terminus

**Comment by dustovich1** [comment]
As a workaround for 1.0.114 there is a UAC.exe in the `%USERPROFILE%\AppData\Local\Programs\Terminus\resources\extras\ssh-keygen` folder that does not require debug DLLs. I just copied it from there to the previous folder and everything works fine again.

**Comment by Eugeny** [comment]
Fixed in 1.0.115
